independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test/multipart/OtherMultipartResource.java

```diff
@@ -15,6 +15,7 @@ public class OtherMultipartResource {
     @Produces(MediaType.TEXT_PLAIN)
     @Consumes(MediaType.MULTIPART_FORM_DATA)
     @POST
+    @SuppressWarnings({ "removal" })
     public String simple(@MultipartForm OtherFormData formData) {
```

Comment:
How about removing this annotation for 3.0 and including an openrewrite rule that converts this into a `@BeanParam`?

Reply:
I don't want to do that just yet because the annotation was only deprecated a couple versions ago

Reply:
Sure, just suggesting because migrating to a major version is handled by an update script and shouldn't (at least) break user code.

Reply:
Right. If we had deprecated at around 2.10, I'd be fine with removing it, but that's not the case unfortunately

Reply:
Note that the fact that we don't remove it shouldn't prevent us from including it into the migration rules (if the transformation is simple enough).

With 3, most people will use our migration tool, so it's a good opportunity to modernize their applications in passing.

Let me know if we need to add something (see https://github.com/quarkusio/quarkus-updates/blob/main/recipes/src/main/resources/quarkus-updates/core/3alpha.yaml#L1026-L1043).

Reply:
Here it is: https://github.com/quarkusio/quarkus-updates/pull/9